shim/opentelemetry-opentracing-shim/tests/testbed/test_common_request_handler/test_asyncio.py

```diff
@@ -129,9 +129,9 @@ async def do_task():
         spans = self.tracer.finished_spans()
         self.assertEqual(len(spans), 3)
 
-        spans = sorted(spans, key=lambda x: x.start_time)
```

Comment:
on windows the parent span has the same start time as the child so sorting does not result in the first span being the parent.

Reply:
Added a comment to the sort convenience method provided by base test case.

Reply:
LMK if the comment is good. I can update in another PR is not.